Pinta.yaml

```diff
@@ -1,14 +1,24 @@
 name: Pinta
 sources:
+  - nuget_sources.json
   - type: git
     url: https://github.com/PintaProject/Pinta.git
-    tag: "1.7"
-    commit: 3d6c188680a389c1f9f40138c092b05ba4009b2e
-  # https://github.com/PintaProject/Pinta/pull/140
-  # https://github.com/PintaProject/Pinta/pull/142
-  # https://github.com/PintaProject/Pinta/pull/145
-  - type: patch
-    path: pinta-1.7-appdata.patch
+    #tag: "1.8"
+    #commit: 3d6c188680a389c1f9f40138c092b05ba4009b2e
+    branch: "gtk3-v2"
```

Comment:
please pin a commit. A bare branch doesn't lead to a reproducable build. (and comment off branch so that it's not breaking either)

Reply:
I am waiting for the release of Pinta 1.8 to do exactly this. I kept the comments for a reminder.

Reply:
FYI the gtk3-v2 branch was recently merged into master, and was upgraded to .NET 6 so a couple updates might be needed here.

Reply:
@cameronwhite Thanks for your info. This makes implementing such changes a lot easier when I don't have to figure everything out on my own! See #25 for the changes.